### ISSUE TITLE
Question about training the generator

### ISSUE THREAD

**Issue by uridabomb** [facebookresearch/low-shot-shrink-hallucinate#7]
Hi, I'm trying to use the method on a completely different dataset. My implementation is in Keras and I built the generator model such that I can view the two losses (rms loss and cls loss) during the training phase. I trained for 10 epochs and I get suspicious results:

1. The classification loss/accuracy does not change at all (is it because I don't change the weights of the classifier?) and the accuracy is *very* small: `~0.0818` (the classifier achieved `~0.7` accuracy on a test set for trained categories).
2. The rms loss/accuracy improves - from `~0.6390` accuracy in the first epoch to `~0.7245` in the last epoch while I used `λ=1`.
3. The rms loss/accuracy converges to `~0.6286` in the second epoch (and does not change at all aftwards) while I used `λ=10` (as you did).

Is there something I should be worried about those results? 
Do you have any enlightenments?

Thanks!
